src/pages/home/report/ReportActionCompose.js

```diff
@@ -1197,20 +1223,25 @@ class ReportActionCompose extends React.Component {
                             // Keep focus on the composer when Send message is clicked.
                             onMouseDown={(e) => e.preventDefault()}
                         >
-                            <Tooltip text={this.props.translate('common.send')}>
-                                <PressableWithFeedback
-                                    style={[styles.chatItemSubmitButton, this.state.isCommentEmpty || hasExceededMaxCommentLength ? undefined : styles.buttonSuccess]}
-                                    onPress={this.submitForm}
-                                    disabled={this.state.isCommentEmpty || isBlockedFromConcierge || this.props.disabled || hasExceededMaxCommentLength}
```

Comment:
Hmm why was this removed? It looks like removing this is causing https://github.com/Expensify/App/issues/23848

Reply:
It's moved here - 

https://github.com/Expensify/App/blob/9e4deb5720e74e97d583303f5000adb7e72057ff/src/pages/home/report/ReportActionCompose.js#L994